ui/explore/src/views/ViewExplore/ViewExplore.tsx

```diff
@@ -0,0 +1,74 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Box, BoxProps } from '@mui/material';
+import {
+  TimeRangeProviderWithQueryParams,
+  useInitialRefreshInterval,
+  useInitialTimeRange,
+} from '@perses-dev/plugin-system';
+import { DEFAULT_DASHBOARD_DURATION, DEFAULT_REFRESH_INTERVAL, DashboardResource } from '@perses-dev/core';
+import { ErrorAlert, ErrorBoundary, combineSx } from '@perses-dev/components';
+import {
+  DatasourceStoreProviderProps,
+  TemplateVariableProviderProps,
+  DatasourceStoreProvider,
+  DashboardProvider,
+  TemplateVariableProvider,
+} from '@perses-dev/dashboards';
+import { ViewExploreApp } from './ViewExploreApp';
+
+export interface ViewExploreProps extends Omit<BoxProps, 'children'> {
+  dashboardResource: DashboardResource;
+  datasourceApi: DatasourceStoreProviderProps['datasourceApi'];
+  externalVariableDefinitions?: TemplateVariableProviderProps['externalVariableDefinitions'];
+  exploreTitleComponent?: JSX.Element;
+}
+
+export function ViewExplore(props: ViewExploreProps) {
+  const { dashboardResource, datasourceApi, externalVariableDefinitions, sx, exploreTitleComponent, ...others } = props;
+
+  const initialTimeRange = useInitialTimeRange(DEFAULT_DASHBOARD_DURATION);
+  const initialRefreshInterval = useInitialRefreshInterval(DEFAULT_REFRESH_INTERVAL);
+
+  return (
+    <DatasourceStoreProvider dashboardResource={dashboardResource} datasourceApi={datasourceApi}>
+      <DashboardProvider initialState={{ dashboardResource, isEditMode: true }}>
+        <TimeRangeProviderWithQueryParams
```

Comment:
We should probably make a wrapper provider component at some point that encapsulates the common providers like TimeRangeProvider, TemplateVariableProvider, etc.

Reply:
So something like this, but a first class version exported out from dashboards package:
https://github.com/sjcobb/perses-embed-example/blob/main/components/dashboards/PersesDashboardProvidersAlt.tsx#L38

Reply:
I almost sure @AntoineThebaud or @Gladorme is working in something like that

Reply:
I think you meant @celian-garcia here, he was working on refactorings around those providers to facilitate the embed use cases, e.g https://github.com/perses/perses/pull/1475

Reply:
Yeah I'm not sure what would be the best between a wrapper or more fine tuned provider asking less mandatory parameters sometimes not useful. 

e.g here the DatasourceStoreProvider ask for a dashboard which should be optional for sure. Then a dashboardProvider is necessary ? maybe this shouldn't